src/service/data_products/common_models.py

```diff
@@ -82,10 +81,10 @@ class DataProductMissingIDs(SubsetProcessStates):
     )
 
 
-QUERY_VALIDATOR_LOAD_VERSION_OVERRIDE = Annotated[str | None, Query(
+QUERY_VALIDATOR_LOAD_VERSION_OVERRIDE = Annotated[str, Query(
```

Comment:
For `Path` and `Query` components, adding `None` to the type annotation causes the input restrictions not to be shown in the OpenAPI documentation. Not sure why

Reply:
Does it happen with a Optional[Str]  Or Union[] ?

Reply:
I didn't try because the current way works well enough for now and I don't want to go back to the Bad Old Ways of type hints